lib/statusbar-summary-view.coffee

```diff
@@ -27,16 +27,24 @@ class StatusBarSummaryView
     @tile = statusBar.addRightTile({item: el, priority: 100})
 
 class StatusBarSummary extends View
+  initialize: ->
+    @on 'click', '.linter-summary-container', ->
+      console.log("moveToNextMessage")
```

Comment:
Hey @ababol :smile: 

I'm cool with moving it to utils, though with more thought perhaps we could come up with something better. I'm wondering if there should be some kind of linter model instance that the views talk to. But I don't have a concrete suggestion at the moment. 

I encourage you to experiment with how you think we should hook this up, and report back!

Also, I noticed #424. Perhaps it's orthogonal to the thing you are working on, but I wanted to make you aware of it in case it turns out to be related.